.registry/resources/oam.stacks.crossplane.io_definitionsinstances.crd.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: definitionsinstances.oam.stacks.crossplane.io
+spec:
+  preserveUnknownFields: true
+  group: oam.stacks.crossplane.io
+  names:
+    kind: DefinitionsInstance
```

Comment:
How would you feel about this just being named `Definitions` or even `OAMDefinitions` (despite the latter being a bit redundant)? I think the trend of naming our CRs `<T>Instance` came from services like `RDS`, `CloudSQL` etc where the cloud provider conceptually considers the resource an "instance" - i.e. a VM doing a thing. We've since added CRs like `WordpressInstance`, which I presume means "an instance of wordpress", as in "one deployment of wordpress". This feels a little redundant to me, as almost all Kubernetes resources are inherently "an instance of" some concept. `Deployment` is not named `DeploymentInstance`, for example.

Reply:
Yeah I originally had it as just `Definitions` but mostly just changed it for the singular and plural to be different. Happy to change it back if you like.